.travis.yml

```diff
@@ -4,6 +4,11 @@ node_js:
   - 10
 git:
   depth: 5
+before_script:
+  # https://github.com/lerna/lerna/issues/1457#issuecomment-399678173
+  - yarn lerna exec yarn install
+  - yarn lerna link --force-local
```

Comment:
Maybe it's a Lerna + Linux issue, but `lerna bootstrap` seems to work only partially on Travis, while `lerna exec yarn install` works entirely. https://github.com/lerna/lerna/issues/1457#issuecomment-399678173

I'm adding this workaround for now, but I'll also report this issue to Lerna to see if they can figure it out.